assets/js/editor-components/sidebar-compatibility-notice/style.scss

```diff
@@ -0,0 +1,13 @@
+.wc-blocks-sidebar-compatibility-notice {
+	border-top: 1px solid $gray-200;
+}
+
+.wc-blocks-sidebar-compatibility-notice__notice.components-notice.is-dismissible {
+	margin: 0;
+	padding: $gap;
+	padding-right: $gap-small;
+
+	.components-notice__content {
+		margin-right: 0;
```

Comment:
This style isn't applied on my end!

<img width="233" alt="image" src="https://user-images.githubusercontent.com/14235870/183071019-0d166dde-d5ee-40b6-beab-c47f0ae4622a.png">